package.json

```diff
@@ -43,11 +43,11 @@
     "postinstall": "patch-package"
   },
   "devDependencies": {
-    "@babel/core": "^7.20.12",
-    "@babel/node": "^7.20.7",
-    "@babel/preset-env": "^7.16.4",
-    "@babel/preset-react": "^7.16.0",
-    "@babel/preset-typescript": "^7.18.6",
+    "@babel/core": "^7.23.3",
+    "@babel/node": "^7.22.19",
+    "@babel/preset-env": "^7.23.3",
+    "@babel/preset-react": "^7.23.3",
+    "@babel/preset-typescript": "^7.23.3",
```

Comment:
Had to bump versions to get proper support for TS `satisfies`, particularly in `jest` (which uses Babel under the hood to transpile `.tsx` files).